packages/action-merge/src/components/MergeWizard/MergeResponse.js

```diff
@@ -39,6 +38,19 @@ EntityResponseTable.propTypes = {
   responseEntities: React.PropTypes.array.isRequired
 }
 
+export const PermissionMessageBox = props => {
+  if (!props.showPermissionMessage || props.showPermissionMessage === false) return <div/>
```

Comment:
|| props.showPermissionMessage === false is not needed, !props.responseEntities covers all cases